code/lora/lora.py

```diff
@@ -0,0 +1,298 @@
+##############################
+# WORKS ONLY WITH PYTHON 3.7 #
+##############################
+
+import os
+import time
+#TODO import argparse
+from pathlib import Path
+from SX127x.LoRa import *
+from SX127x.board_config import BOARD
+
+HOME_PATH = Path.home()
+LORA_LOGS = HOME_PATH / 'lora_logs'
+RX_LOG = LORA_LOGS / 'rx.txt' #File, where all received data is stored
+STATUS_LOG = LORA_LOGS / 'status.txt' #File, where this block's behavior is logged
+CONF_FILE = LORA_LOGS / 'conf.txt' #File, where commands for this block are issued by other scripts
+IMAGES = HOME_PATH / 'images'
+THERMAL = HOME_PATH / 'thermal'
+
+#Log files with data to be sent
+#TODO: Argparse or .conf of some sort
+TX_FILES = [
+    HOME_PATH / 'sensor_logs' / 'alpha.txt',
+    HOME_PATH / 'sensor_logs' / 'bravo.txt',
+    HOME_PATH / 'sensor_logs' / 'charlie.txt'
+]
+
+IMAGES_PATH = HOME_PATH / 'images'
+THERMAL_PATH = HOME_PATH / 'thermal'
+
+UP_PACKET_LENGTH = 11 #Base -> Sat packet length in bytes
```

Comment:
We should store all these constants in some config file, I suggest using [yaml](https://pl.wikipedia.org/wiki/YAML) for that, it's super easy to read by humans.
It is easy like:
```python
import yaml
config_dict = yaml.loads(open('file.yaml', 'r').read())
```

Reply:
Would configparser and config.ini be fine too?

Reply:
Congifparser is fine. I'm just feel more comfortable with yaml "markdown" than weird configparser syntax, that's why I suggested that. Configparser it is then!